gwtp-core/gwtp-dispatch-rest/src/test/java/com/gwtplatform/dispatch/rest/rebind/parameter/CookieParamValueResolverTest.java

```diff
@@ -29,7 +29,7 @@
 public class CookieParamValueResolverTest {
     private static final String SOME_VALUE = "llamas";
 
-    private CookieParamValueResolver resolver = new CookieParamValueResolver();
+    private final CookieParamValueResolver resolver = new CookieParamValueResolver();
```

Comment:
Don't bother. I just actually don't care whether test fields are final or not, but might as well keep it now ;)